include/operations/blas2_trees.h

```diff
@@ -26,27 +26,44 @@
 #ifndef SYCL_BLAS_BLAS2_TREES_H
 #define SYCL_BLAS_BLAS2_TREES_H
 namespace blas {
-template <typename output_t, typename matrix_t, typename vector_t>
+template <typename lhs_t, typename matrix_t, typename vector_t,
+          bool is_transposed, int cache_line_size, int work_per_thread>
 struct Gemv {
   using value_t = typename vector_t::value_t;
   using index_t = typename vector_t::index_t;
-  output_t lhs_;
-  matrix_t matrix_;
-  vector_t vector_;
+  lhs_t lhs_;
+  matrix_t matrix_a_;
+  vector_t vector_x_;
+  index_t WGs_per_NC_;
```

Comment:
can you follow the same name convention as other files (AFAIK we don't use capital naming for class members)

Reply:
Updated